api/app/utils/fba_calculator.py

```diff
@@ -12,17 +15,14 @@ class CannotCalculateFireTypeError(Exception):
 
 
 class FBACalculatorWeatherStation():  # pylint: disable=too-many-instance-attributes
-    """ A combination of station data from WFWX API and user-specified inputs for
-    Fire Behaviour Advisory Calculator """
+    """ Inputs for Fire Behaviour Advisory Calculator """
 
     def __init__(self,  # pylint: disable=too-many-arguments
-                 wfwx_id: str, code: int, elevation: int, fuel_type: str,
+                 elevation: int, fuel_type: str,
                  time_of_interest: date, percentage_conifer: float,
                  percentage_dead_balsam_fir: float, grass_cure: float,
-                 crown_base_height: int, lat: float, long: float, name: str):
-        self.wfwx_id = wfwx_id
-        self.code = code
-        self.name = name
+                 crown_base_height: int, lat: float, long: float, bui: float, ffmc: float, isi: float,
```

Comment:
didn't we want to keep the name in the response? Doesn't that make things easier on the FE?

Reply:
It's still in the response. It's now set in the "prepare_response" function.